Source/Atomic/IO/Log.cpp

```diff
@@ -228,10 +228,10 @@ void Log::WriteRaw(const String& message, bool error)
     if (logInstance->quiet_)
     {
         if (error)
-            __android_log_print(ANDROID_LOG_ERROR, "Urho3D", message.CString());
+            __android_log_print(ANDROID_LOG_ERROR, "Atomic", "%s", message.CString());
     }

```

Comment:
Maybe pass this fix upstream, otherwise good idea to add an // ATOMIC BEGIN/END here with description of change, helps to not clobber changes when pulling updates